dist/Makefile

```diff
@@ -1,15 +1,15 @@
 HOSTPORT = 12345
 IMAGENAME = "malibu"
 
-.PHONY: docker run-docker
+.PHONY: docker docker-run docker-run-%
 
 docker:
 	docker build -t $(IMAGENAME) ..
 
-run-docker: docker
+docker-run: docker
 	@echo "Running 'malibu' container on port $(HOSTPORT)"
-	@echo "Open 'http://localhost:$(HOSTPORT)/isbn/suche.html' to access it"
-	docker run --rm -it -p $(HOSTPORT):80 "$(IMAGENAME)"
+	@echo "Open 'http://localhost:$(HOSTPORT)/malibu/isbn/suche.html' to access it"
```

Comment:
Okay I did, but I doubt that it will help. I just put the `@echo localhost:port/path` in there to remind myself. Users that want to deploy the latest version should use the `ubma/malibu` image and won't see the message. Documentation in INSTALL.md / README.md makes more sense IMHO.